coffee-shop-application/src/main/kotlin/com/motycka/edu/order/OrderService.kt

```diff
@@ -0,0 +1,112 @@
+package com.motycka.edu.order
+
+import com.motycka.edu.customer.CustomerRepository
+import com.motycka.edu.menu.MenuItemDTO
+import com.motycka.edu.menu.MenuItemResponse
+import com.motycka.edu.menu.MenuRepository
+
+class OrderService(
+    private val orderRepository: OrderRepository,
+    private val orderItemRepository: OrderItemRepository,
+    private val menuRepository: MenuRepository,
+    private val customerRepository: CustomerRepository
+) {
+
+    suspend fun createOrder(userId: Long, request: OrderRequest): OrderResponse {
+        val customer = customerRepository.selectCustomer(userId)
+            ?: throw IllegalArgumentException("Customer not found")
```

Comment:
Tip: In kotlin, `error` function produces IllegalArgumentException. Although it has the same result, the function is more idiomatic.